src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -972,11 +979,21 @@ public AppPage verifyContains(String searchString) {
         return this;
     }
 
-    public void verifyContainsElement(By by) {
-        List<WebElement> elements = browser.driver.findElements(by);
+    public void verifyContainsElement(By childBy) {
+        List<WebElement> elements = browser.driver.findElements(childBy);
         assertFalse(elements.isEmpty());
```

Comment:
Can just inline `elements` here? The method can be one line long. Same for the other two methods.